kcidb/monitor/subscriptions/linux_stable_rc.py

```diff
@@ -19,6 +19,14 @@ def match_revision(revision):
     if revision.builds_valid is None:
         return ()
 
+    # If the revision is not from 'maestro' or 'broonie' origin
+    origins = []
+    for _, branch_checkouts in revision.repo_branch_checkouts.items():
+        for _, checkouts in branch_checkouts.items():
+            origins.extend([checkout.origin for checkout in checkouts])
```

Comment:
You can just iterate over `revision.checkouts` and simplify this code. Down to something like this:
```python
if not {c.origin for c in revision.checkouts} & {'maestro', 'broonie'}:
    return ()
```

Reply:
Oh, yes. I'll update it.